app/components/troubleshooting_options_component.rb

```diff
@@ -2,14 +2,19 @@ class TroubleshootingOptionsComponent < BaseComponent
   renders_one :header, 'TroubleshootingOptionsHeadingComponent'
   renders_many :options, BlockLinkComponent
 
-  attr_reader :tag_options
+  attr_reader :options_from_constructor, :tag_options
 
-  def initialize(**tag_options)
+  def initialize(options: [], **tag_options)
+    @options_from_constructor = options
     @tag_options = tag_options.dup
   end
 
+  def all_options
+    options_from_constructor + options
+  end
```

Comment:
It might not be too bad, actually:

https://github.com/ViewComponent/view_component/blob/fff1d586c1a5450479738ed30402822fcc73ccf0/lib/view_component/slotable.rb#L91-L97

Something like this, perhaps...

```rb
def options
  options_from_constructor + get_slot(:options)
end

def options?
  options.present?
end
```

Reply:
Updated in 7000581ef.